.test-infra/metrics/README.md

```diff
@@ -17,30 +17,19 @@
     under the License.
 -->
 # BeamMonitoring
-This folder contains resources required to deploy the Beam metrics stack.
-There are two types of metrics in Beam project:
-* Community metrics
-* Metrics published by tests (IO Performance tests, Load tests and Nexmark tests) 
+This folder contains resources required to deploy the Beam community metrics
+stack.
 
-Both types of metrics are presented in [Grafana dashboard available here.](https://s.apache.org/beam-community-metrics)
+[Beam community dashboard is available here.](https://s.apache.org/beam-community-metrics)
```

Comment:
Please, update link to metrics.beam.apache.org

Reply:
Does this link work? I'm getting a timeout. The current link (https://s.apache.org/beam-community-metrics) is OK, though.

Reply:
@Ardagan

Reply:
This is a link I use most of the time:
http://metrics.beam.apache.org

Reply:
It works now. I had to add an exclusion to my HTTPS Everywhere browser plugin.

Reply:
I think I'll update the link in the next PR. This PR is just a revert and it might be good not to add more commits here. Does this sound good?